rest.php

```diff
@@ -95,6 +95,10 @@
             $controller->groups($request);
         } elseif ($parts[2] == 'groupMembers') {
             $controller->groupMembers($request);
+        } elseif ($parts[2] == 'newConcepts') {
```

Comment:
This is a matter of taste but I think `new` and `modified` would be better REST method names. Also, `new` matches with the URL of the corresponding web page (e.g. http://finto.fi/yso/en/new)